lib/worker/fake-tty.js

```diff
@@ -4,6 +4,7 @@ const options = require('./options').get();
 
 if (options.tty) {
 	Object.assign(process.stdout, {isTTY: true}, options.tty);
+	process.stdout.getColorDepth = () => options.tty.colorDepth;
```

Comment:
We should only shim this in Node.js versions where it's expected to be available.